lib_test/main.ml

```diff
@@ -22,11 +30,8 @@ let cmd =
   let man = [
     `P "Requires a running xenstored to work. See github.com:mirage/ocaml-xenstore-server"
   ] in
-  Term.(ret (const test $ socket)),
-  Term.info "test_ezxenstore" ~doc ~man
-
-let () = match Term.eval cmd with `Ok x -> exit x | _ -> exit 1
-
-
-
+  Cmd.v
+    (Cmd.info "test_ezxenstore" ~doc ~man)
```

Comment:
Is this the name of the binary? The `dune` files does not contain a `public_name`